src/Sulu/Bundle/AdminBundle/Resources/js/components/MultiItemSelection/README.md

```diff
@@ -0,0 +1,74 @@
+
+The `MultiItemSelection` component is a list used for referencing different datasets in `Sulu`. Inside the 
+`MultiItemSelection` those references can be added, sorted and deleted. The sorting can be done by drag and drop.
+
+```
+const arrayMove = require('react-sortable-hoc').arrayMove;
```

Comment:
Sorry, didn't I write that we should export exactly this `arrayMove` function in `components/index.js`?